security-command-center/snippets/src/test/java/vtwo/FindingsIT.java

```diff
@@ -130,4 +134,13 @@ public void testGroupFilteredFindings() throws IOException {
         LOCATION);
     assertThat(stdOut.toString()).contains("count: 1");
   }
+
+  @Test
+  public void testSetFindingsByStateInactive() throws IOException {
+    SetFindingsByState.setFindingState(ORGANIZATION_ID, SOURCE.getName().split("/")[3],
+        LOCATION, FINDING_1.getName().split("/")[7]);
+    assertThat(stdOut.toString()).contains("Updated Finding: ");
```

Comment:
add a line before this assert. Please follow the guidelines from http://go/unit-testing-practices#structure